proto/cometbft/types/v1/params.proto

```diff
@@ -17,10 +17,8 @@ message ConsensusParams {
   EvidenceParams  evidence  = 2;
   ValidatorParams validator = 3;
   VersionParams   version   = 4;
-  // TODO: move ABCI to FeatureParams
-  ABCIParams      abci      = 5;
-  SynchronyParams synchrony = 6;
-  FeatureParams   feature   = 7;
+  SynchronyParams synchrony = 5;
+  FeatureParams   feature   = 6;
```

Comment:
Does not matter really, `ABCI` has never been introduced in a stable release.

Reply:
Are you sure? Vote extensions are available in `v0.38.x`.

Reply:
@cason is right

The way we managed this when introducing ABCI++ was to deprecate the old field numbers rather then reusing them, to avoid (unlikely but unfortunate) confusion of fields. I see that @mzabaluev kept that way of deprecating fields when he introduced versioning.
So I'd do the same here: I would leave `FeatureParams` as 6, set `SynchronyParams` to 7, and declare 5 as reserved. For an example, take a look at this [message definition](https://github.com/cometbft/cometbft/blob/2c6e0c160/proto/cometbft/abci/v1/types.proto#L37)

Reply:
Ah yes, I got confused by `cometbft.types.v1` which also applies back to 0.38.

In this case, you could only make field 5 wire-compatible if you use exactly this field definition in `FeatureParams`:

```proto
int64 vote_extensions_enable_height = 1;
```